gencert.sh

```diff
@@ -11,3 +11,7 @@ docker run --rm --volume $PWD/certs:/tmp --user $(id -u):$(id -g) eventstore/es-
 docker run --rm --volume $PWD/certs:/tmp --user $(id -u):$(id -g) eventstore/es-gencert-cli:1.0.2 create-node -ca-certificate /tmp/ca/ca.crt -ca-key /tmp/ca/ca.key -out /tmp/node -ip-addresses 127.0.0.1 -dns-names localhost
 
 chmod -R 0755 ./certs
+
+cp certs/ca/ca.crt /usr/local/share/ca-certificates/eventstore_ca.crt
+
+sudo update-ca-certificates
```

Comment:
I believe we may need to check if the host is running macOS as these commands won't work on macOS, at least the way we want them to.

The first one will, but it will be copying the certificate to a redundant location (needs to go into Keychain and then trust using `security` command)

Reply:
The tests are run on Ubuntu only, except for test/EventStore.Client.Tests (which doesn't require certs). However, I added them anyway for convenience. I also included the script for Windows.